src/App/hooks/useFlag/useFlag.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import style from './useFlag.module.scss';
+
+const useFlag = () => {
+  function codeToCountryFlag(iso2: string) {
```

Comment:
let's rename function to "flag" and write something like this
`const flag = (iso2: string) => {...}`